build.sbt

```diff
@@ -26,6 +26,13 @@ lazy val `metaconfig-docs` = project
   )
   .dependsOn(`metaconfig-coreJVM`)
 
+lazy val `metaconfig-schema` = project
```

Comment:
Let's call this `metaconfig-json` and I will take care of moving the ujson parts into this module.